packages/core/test/use-router-params.test.ts

```diff
@@ -0,0 +1,29 @@
+import {exportRouterParams} from '@blitzjs/core'
+
+describe('useRouterParams', () => {
+  describe('exportRouterParams', () => {
+    it('returns proper params', () => {
+      const routerQuery = {
+        id: '1',
+        cat: 'category',
+        slug: ['example', 'multiple', 'slugs'],
+        empty: '',
+        queryArray: ['1', '123', ''],
+      }
+
+      const query = {
+        cat: 'somethingelse',
+        slug: ['query-slug'],
+        queryArray: ['1', '123', ''],
```

Comment:
There's one more case which isn't covered here which is the result `params` should not include a key that is in `query`, but not in `routerQuery`. So I think just add a key here to `query` and tests should still pass.

Makes sense? :)